apps/site/assets/css/_select-custom.scss

```diff
@@ -0,0 +1,38 @@
+.c-select-custom {
```

Comment:
in an earlier version of this code I had made the custom select CSS more module so it can be re-used... but then I didn't end up re-using it.  I could remove this... or make it it's own small PR... but I consider it to be fairly harmless, so I am leaving it in for now :)